src/store/reducers/cohorts.reducer.test.js

```diff
@@ -7,7 +7,7 @@ const initialState = {
 
 describe('Cohorts Reducer', () => {
   it('returns initial state', () => {
-    expect(cohortsReducer(undefined, {})).toEqual(initialState);
+    expect(cohortsReducer(undefined, {})).toEqual({});
```

Comment:
I think once the reducers are in separate files with separate tests and INITIAL_STATE in cohorts.reducer.js is reverted, this can be changed back to initialState.